pkg/signature/kms/cliplugin/common/interface.go

```diff
@@ -26,8 +26,7 @@ const (
 	// Breaking changes to the PluginClient and this schema necessarily mean major version bumps of
 	// this ProtocolVersion and the sigstore version.
 	// Plugin authors may choose to be backwards compatible with older versions.
-	// TODO: change this to be semver compatible, like v1, or v1.0.0.
-	ProtocolVersion               = "1"
+	ProtocolVersion               = "v1.0.0"
```

Comment:
I think this amount of semver specificity is overkill and a single major version should be sufficient. Clients don't need to be aware of non-breaking changes to the protocol.

Reply:
changed